tests/Integration/PrestaShopBundle/Form/TestProductFormType.php

```diff
@@ -47,18 +47,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('stock', FormType::class)
+            ->add('description', FormType::class)
             ->add('shipping', FormType::class)
             ->add('options', FormType::class)
             ->add('pricing', FormType::class)
         ;
+        $description = $builder->get('description');
+        $description->add('packed_products', FormType::class);
```

Comment:
`packed_products` shoud be added to `stock` to reflect its actual position

What's weird is that the listener is correctly setup to handle stock, but the tests don't seem to fail even though there's no reason why this `description.packed_products` field would be removed in any case

Reply:
It should fail in cases related to `description.packed_products` since it's never removed This makes me wonder if the unit test is actually relevant 🤔